Automattic/jetpack
# Contact Form: Update block placeholder

Fixes #27130

#### Changes proposed in this Pull Request:
* Update Form block placeholder to include guidance links

More context on #27130 and [here](pcsBup-lD-p2)

![image](https://user-images.githubusercontent.com/7811225/198408385-0a882524-24b8-43c5-9ad6-124706bcc5c8.png)

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a Post/Page and add a Form block
* Check if the block placeholder contains the changes described above
